src/Castle.Services.Logging.log4netIntegration/ExtendedLog4netFactory.cs

```diff
@@ -16,14 +16,22 @@ namespace Castle.Services.Logging.Log4netIntegration
 {
 	using System;
 	using System.IO;
+	using System.Reflection;
 
 	using Castle.Core.Logging;
 
 	using log4net;
 	using log4net.Config;
 
+
 	public class ExtendedLog4netFactory : AbstractExtendedLoggerFactory
 	{
+#if NET35 || NET40
```

Comment:
Can you change this to `#if FEATURE_LEGACY_REFLECTION_API`.

Reply:
Fixed